src/plugin.js

```diff
@@ -35,13 +38,33 @@ class BlessCSSWebpackPlugin {
                 input.source = asset.source();
               }
 
+              const filenameWithoutExtension = cssFileName.replace(CSS_REGEXP, '');
+
+              /**
+               * Inject @import rules into the primary .css file for all others
+               */
+              if (this.options.importRules) {
+                const parsedDataSimple = bless.chunk(input.source);
+                let importRules = '';
+
+                parsedDataSimple.data.map((fileContents, index) => {
+                  if (index == 0) return;
+
+                  let filename = index === 0 ? cssFileName : `${filenameWithoutExtension}-blessed${index}.css`;
+                  //e.g. @import url(app-blessed1.css);
+                  importRules += '@import url(' + filename + ');\n';
```

Comment:
The filename also needs to respect the [`output.path`](https://webpack.js.org/configuration/output/#output-path) and [`output.publicPath`](https://webpack.js.org/configuration/output/#output-publicpath) webpack options.